src/simtools/applications/generate_simtel_event_data.py

```diff
@@ -1,12 +1,89 @@
 #!/usr/bin/python3
 
 r"""
-Generate a reduced dataset of event data from simulated files.
+Generate a reduced dataset of event data from output of telescope simulations.
 
-Processes sim_telarray output files (typically of type '.simtel.zst') and creates a
-reduced dataset containing shower information, array-level parameters, and data about
+Processes sim_telarray output files (typically of type '.simtel.zst') and creates
+reduced datasets containing shower information, array-level parameters, and data about
 triggered telescopes.
 
+The output consists of an HDF5 or FITS file containing the following tables:
+
+**FILE_INFO**
+
++-------------------+---------+-----------------------------------------------+
+| Field             | Type    | Description                                   |
++===================+=========+===============================================+
+| file_name         | string  | Name of the file                              |
++-------------------+---------+-----------------------------------------------+
+| file_id           | int64   | Internal unique identifier for the file       |
++-------------------+---------+-----------------------------------------------+
+| particle_id       | int64   | PDG particle ID (e.g., 14 for proton)         |
++-------------------+---------+-----------------------------------------------+
+| energy_min        | float32 | Minimum simulated energy (TeV)                |
++-------------------+---------+-----------------------------------------------+
+| energy_max        | float32 | Maximum simulated energy (TeV)                |
++-------------------+---------+-----------------------------------------------+
+| viewcone_min      | float32 | Min viewcone angle (rad)                      |
++-------------------+---------+-----------------------------------------------+
+| viewcone_max      | float32 | Max viewcone angle (rad)                      |
++-------------------+---------+-----------------------------------------------+
+| core_scatter_min  | float32 | Min core scatter radius (m)                   |
++-------------------+---------+-----------------------------------------------+
+| core_scatter_max  | float32 | Max core scatter radius (m)                   |
++-------------------+---------+-----------------------------------------------+
+| zenith            | float32 | Zenith angle (rad)                            |
```

Comment:
Why are all the angles in radians instead of degrees which are more user friendly?